boilerplate/act-rules-format/header.include

```diff
@@ -12,7 +12,7 @@
   <header>
     <p data-fill-with="logo"></p>
     <h1 id="title" class="p-name no-ref">[TITLE] [LEVEL]</h1>
-    <h2 id="profile-and-date" class="no-num no-toc no-ref">[LONGSTATUS],
+    <h2 id="w3c-state" class="no-num no-toc no-ref">[LONGSTATUS],
```

Comment:
While pubrules only check for #w3c-state and our stylesheet only styles #w3c-state, I recommend using a paragraph rather than h2 or h3. h2 or h3 should be reserved for subtitles. This also aligns with [ReSpec](https://github.com/w3c/respec/blob/develop/src/w3c/templates/headers.js#L158).

Reply:
Done.